### PR TITLE
Blockchains > Get Started > Build Custom Blockchains

### PR DESCRIPTION
**Pull request by nhussein11**
This PR creates the "Build Custom Blockchains" page in the `Blockchains > Get Started` section

### CODE REVIEW

File: develop/blockchains/get-started/build-custom-blockchains.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+## Starting from Templates
+
+Using pre-built templates is an efficient way to begin building a custom blockchain. Templates provide a foundational setup with pre-configured modules, letting developers avoid starting from scratch and instead focus on customization. Depending on your project’s goals—whether you want a simple test chain, a standalone chain, or a parachain that integrates with Polkadot’s relay chains—there are templates designed to suit different levels of complexity and scalability.
+
+Within the Polkadot SDK, the following templates are available to get you started:
+
+- [minimal-template](https://github.com/paritytech/polkadot-sdk/tree/master/templates/minimal){target=\_blank} - includes only the essential components necessary for a functioning blockchain. It’s ideal for developers who want to gain familiarity with blockchain basics and test simple customizations before scaling up
```
Comment (CrackTheCode016) [inline develop/blockchains/get-started/build-custom-blockchains.md:20]: Maybe format: `minimal-template`

File: develop/blockchains/get-started/build-custom-blockchains.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- **Test and debug** - testing is essential to ensure your custom chain works as intended. Conduct unit tests for individual pallets and integration tests for interactions between pallets
+
+- **Compile** - after finalizing and testing your custom configurations, compile the blockchain to generate the necessary executable files for running a node. Run the node locally to validate that your customizations work as expected and that your chain is stable and responsive
+
+- **Deploy to a testnet or mainnet** - once testing is complete, deploy your chain to a live environment. Depending on your project’s goals, you can start with a testnet to monitor performance and gather feedback or launch directly on a mainnet
+
+Each of these steps is designed to build on the last, helping ensure that your custom blockchain is functional, optimized, and ready for deployment within the Polkadot ecosystem or beyond.
+
+## Monitoring and Maintenance
```
Comment (CrackTheCode016) [inline develop/blockchains/get-started/build-custom-blockchains.md:56]: Link the monitoring / "General Management" page we have maybe?

**Review by CrackTheCode016 — APPROVED** [review]
LGTM! Few comments but looks overall great 🚀

File: develop/blockchains/get-started/build-custom-blockchains.md (hunk 1 above)
Comment (nhussein11) [reply develop/blockchains/get-started/build-custom-blockchains.md:20]: yeah, it's better that way! fixed! thanksss

File: develop/blockchains/get-started/build-custom-blockchains.md (hunk 2 above)
Comment (nhussein11) [reply develop/blockchains/get-started/build-custom-blockchains.md:56]: great! added on [b34fec3](https://github.com/polkadot-developers/polkadot-docs/pull/156/commits/b34fec302747815381ed88074cae9b0ff5f4f73c), thank you!

File: develop/blockchains/get-started/build-custom-blockchains.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+## High-Level Steps to Build a Custom Chain
+
+To build a custom blockchain using the Polkadot SDK, there exist some general steps to follow:
+
+Building a custom blockchain with the Polkadot SDK involves several core steps, from environment setup to deployment. Here’s a breakdown of each stage:
+
+- **Set up the development environment** - install Rust and configure all necessary dependencies to work with the Polkadot SDK (for more information, check the [Install Polkadot SDK dependencies](TODO:update-path){target=\_blank} page). Ensuring your environment is correctly set up from the start is crucial for avoiding compatibility issues later
+
+- **Clone the chain template** - start by downloading the code for one of the pre-built templates that best aligns with your project needs. Each template offers a different configuration, so select one based on your chain’s intended functionality and scalability requirements
```
[the remark this answers is not in the document]
Comment (nhussein11) [reply develop/blockchains/get-started/build-custom-blockchains.md:45]: yeah makes sense, will get rid of the last part, ty!

File: develop/blockchains/get-started/build-custom-blockchains.md [hunk 4]
```diff
@@ -50,10 +50,10 @@ Building a custom blockchain with the Polkadot SDK involves several core steps,
...
 - **Compile** - after finalizing and testing your custom configurations, compile the blockchain to generate the necessary executable files for running a node. Run the node locally to validate that your customizations work as expected and that your chain is stable and responsive
 
-- **Deploy to a testnet or mainnet** - once testing is complete, deploy your chain to a live environment. Depending on your project’s goals, you can start with a testnet to monitor performance and gather feedback or launch directly on a mainnet
-
 Each of these steps is designed to build on the last, helping ensure that your custom blockchain is functional, optimized, and ready for deployment within the Polkadot ecosystem or beyond.
 
-## Monitoring and Maintenance
+## Where to Go Next
+
+Once your chain is functional locally, depending on your project’s goals, you can deploy to a TestNet to monitor performance and gather feedback or launch directly on a MainNet. To learn more about this process, check the [Deployment](){target=\_blank} section of the documentation.
```
Comment (0xLucca) [inline develop/blockchains/get-started/build-custom-blockchains.md:57]: Missing URL in `Deployment`. Fix that and that's all

**Review by 0xLucca — APPROVED** [review]

**Pull request merged**